pandaharvester/harvesterbody/worker_adjuster.py

```diff
@@ -51,6 +51,25 @@ def __init__(self, queue_config_mapper):
             tmp_log.warning("default no_pilots_when_no_active_jobs = False")
             self.no_pilots_when_no_active_jobs = False
 
+    # get queue noPilotsWhenNoActiveJobs
+    def get_queue_no_pilots_when_no_active_jobs(self, site_name=None):
+        tmp_log = core_utils.make_logger(_logger, f"site={site_name}", method_name="get_queue_no_pilots_when_no_active_jobs")
+        ret_val = False
+
+        if self.no_pilots_when_no_active_jobs:
+            return True
+
+        try:
+            if self.queue_configMapper.has_queue(site_name):
+                queue_config = self.queue_configMapper.get_queue(site_name)
+                ret = str(queue_config.submitter.get("noPilotsWhenNoActiveJobs", "false"))
+                if ret.lower() == 'true':
```

Comment:
Why does it have to handle strings like "true" and "false"? Can't it just be bool coming from JSON config like: `"noPilotsWhenNoActiveJobs": true`

Reply:
yes. I can use only bool value. Here just for cases that some users define it with "true".

Reply:
I updated it to `ret_val = bool(queue_config.submitter.get("noPilotsWhenNoActiveJobs", False))`